src/l10n/en.po

```diff
@@ -56,6 +60,18 @@ msgstr ""
 msgid "Go back"
 msgstr ""
 
+#: 7bdc
```

Comment:
Why are these translation values commented out ?

Reply:
In the initial design, the text was used directly, so when I used the command `npx @bluecateng/l10n-cli` for translation, the file was saved with the translated text. However, after the restructuring, the text was no longer used directly, and the command `npx @bluecateng/l10n-cli` these translation values commented out.
From: ``` t`Go to BAM` ```
To: ``` t`${item.text}` ```